### PR TITLE
Made '--debug-brk' parameter optional to be able run testcafe on Electron > 19

### PR DESCRIPTION
**Pull request by lenchvolodymyr**
`--debug-brk` was deprecated since [nodejs 7.7.0](https://nodejs.org/en/docs/guides/debugging-getting-started/#legacy-debugger) and in [electron 1.7.4](https://www.electronjs.org/releases/stable?version=1&page=5#1.7.4) was added a support for `--inspect`. In Electron 19 `--debug-brk` was supressed and no longer works, which leads to an error: "Unable to connect". 

I added a new option `enableDebug` that allows using an old parameter `--debug-brk` for Electron < 1.7.4 and use `--inspect` by default for newer Electron versions.

Related issue: #90

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,6 +175,10 @@ __Optional.__ `testcafe-browser-provider-electron` suppresses `did-navigate` and
 
 __Optional.__ If `true`, DevTools will be opened just before tests start.
 
+### enableDebug
```
Comment (miherlosev) [inline README.md:178]: I don't think we need an extra option.
You can calculating the module logic depending on Electron version:
* if Electron < 19, use the old behavior (the `--debug-brk=<port>` will be used`).
* if Electron >= 19, use the  `--inspect-brk=${ports[1]}` parameter.

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lenchvolodymyr) [reply README.md:178]: I thought about it, but how can I give the electron version?

Comment (miherlosev) [reply README.md:178]: We have a few ideas on how to implement it in another way. We will check them and update this thread as soon we have any news.

Comment (vlad-dargel) [reply README.md:178]: @miherlosev any news here?

Comment (lenchvolodymyr) [reply README.md:178]: if someone needs this update urgently, you can install it from fork, but hope changes will be released soon 
`npm i github:hackolade/testcafe-browser-provider-electron#feat/v0.0.19`

**Comment by lenchvolodymyr** [comment]
@AndreyBelym it's done

**Review by AndreyBelym — APPROVED** [review]

**Pull request merged**